Replicator/DBWorker.cc

```diff
@@ -208,7 +208,7 @@ namespace litecore { namespace repl {
     string DBWorker::effectiveRemoteCheckpointDocID(const C4UUID *localUUID, C4Error *err) {
         // Derive docID from from db UUID, remote URL, channels, filter, and docIDs.
         Array channels = _options.channels();
-        Value filter = _options.properties[kC4ReplicatorOptionFilter];
+            Value filter = _options.properties[kC4ReplicatorOptionFilter];
```

Comment:
Indent level is wrong -- make sure there are no tabs, please. Actually looks like there is no changes aside from the indent, so you could remove this diff.